### DISCUSSION TITLE
Changing the Auth redirect link doesn't seem to work for me?

### DISCUSSION THREAD

**Discussion by Yofou** [supabase/supabase#2253] [Questions] [answered]
when I change my supabase `SITE_URL` from `http://localhost:3000/` to `http://localhost:3000/callback/`
and also changed `Additional_Redirect_URLs` to `http://localhost:3000/callback/`. but even when I login in (with google)
it still redirects me to `http://localhost:3000/`. I've made sure I've saved but still having this issue (for around 2 days now).
hopefully someone can help me. I did follow the guides and it does work just redirects me to the wrong url now.

**Comment by pranavbadami** [comment, marked as answer]
Hey! Are you using a `redirectTo` value when you call `supabase.auth.signIn()` as well? in my experience, having the actual domain name as the redirect url in Supabase settings and using `redirectTo` with your domain + route has worked. 

So in my case:
```
const { user, error } = await supabase.auth.signIn(
      {
        provider: "google",
      },
      {
        redirectTo: `http://localhost:3000/auth/permissions`,
      }
    );
 ```
and the Site URL in the supabase setting would be : `http://localhost:3000/`

**Reply by Yofou** [reply]
Ah, did the fixes you suggested & it all works now. thank you so much 👍🏻
